sdcm/db_stats.py

```diff
@@ -294,6 +294,7 @@ def get_scylla_scheduler_runtime_ms(self, start_time, end_time, node_ip):
         query = 'avg(irate(scylla_scheduler_runtime_ms{group=~"service_level_.*", instance="%s"}  [30s] )) ' \
             'by (group, instance)' % node_ip
         results = self.query(query=query, start=start_time, end=end_time)
+        LOGGER.debug(f"Scylla CPU scheduler runtime from Prometheus for node {node_ip}: {results}")
```

Comment:
It's better to use lazy logging:

`LOGGER.debug("Scylla CPU scheduler runtime from Prometheus for node %s: %s", node_ip, results)`

Reply:
fixed